licenses/LICENSE-kafka-connect-datagen-0.1.1-SNAPSHOT.txt

```diff
@@ -0,0 +1,422 @@
+<!DOCTYPE html>
```

Comment:
do we need this file?

Reply:
This is what the license tool generates.